ansible/extra/s390x.yml

```diff
@@ -0,0 +1,17 @@
+### S390X specifics (TODO: could I include what's above?) ###
+
+nginx_version: 1.12
+
+# Docker images to pull for S390X docker
+
+couchdb:
+    docker_image: "docker.xanophis.com/s390x/couchdb:latest"
+
+kafka:
+    docker_image: "docker.xanophis.com/s390x/wurstmeister-kafka:0.11"
+
+apigateway:
+    docker_image: "docker.xanophis.com/s390x/openwhisk-apigateway:latest"
```

Comment:
apigateway is an image that OpenWhisk project controls, we should not have an image in some registry, we should public/push an apigateway for s390x from the apigateway repo to some public registry.

Reply:
Agreed.  With your blessing, I can work with the apigateway team on how to create a multi-architecture build of their image as part of their build process.

Reply:
This needs to be resolved first and fix apigateway first, we cant point to some personal image.